src/DIRAC/ConfigurationSystem/Client/VOMS2CSSynchronizer.py

```diff
@@ -257,7 +259,8 @@ def syncCSWithVOMS(self):
         if not result["OK"]:
             self.log.error("Could not retrieve user information", result["Message"])
             return result
-
+        if getUserErrors := result.get("Errors", []):
```

Comment:
This is currently not working, because of the way return values are done from the `getUsers` function.

Is this something people would be interested to get send via VOMS2CSAgent at all?
I.e. some missing entry like users that didn't link their certificate could come from this.

Reply:
Now the value contains both Users and Errors